ligo/haskell/test/Test/Ligo/BaseDAO/Proposal.hs

```diff
@@ -1269,3 +1273,42 @@ voteOutdatedProposal _ originateFn = do
     advanceTime (sec 25)
     call dao (Call @"Vote") [params]
       & expectCustomErrorNoArg #vOTING_PERIOD_OVER
+
+proposalCreationUnderStorageLimit
+  :: forall caps base m.
+    ( MonadNettest caps base m
+    , HasCallStack
+    )
+  => IsLorentz -> (ConfigDesc ConfigL -> OriginateFn ParameterL m) -> m ()
+proposalCreationUnderStorageLimit  _ originateFn = do
+  ((owner1, _), _, dao, admin) <- originateFn testConfigWithLongVP
```

Comment:
@sras I now re-considered this approach and I think we should put the proposals in the contract at origination instead.

So, what I think this tests should do is:
1. create a DAO that contains 1000 proposals from the very start
2. check that we can make a new proposal on that
3. check that we can migrate after that as well

My reasoning is that we want to check for a lot more (1000) to guarantee some margine of error and after that we want to check a big operation (`propose`) as well as a `migrate`, which in the worst case scenario should still always work. What do you think?

Note that this way we should also be able to run this test in a reasonable amount of time on the network; the even larger origination will take a long time, but less than making all of the `propose` calls as it is now.